packages/cadl-rlc-emitter/src/transformPaths.ts

```diff
@@ -0,0 +1,43 @@
+import { Paths } from "@azure-tools/rlc-codegen";
+import { Program } from "@cadl-lang/compiler";
+import { getAllRoutes } from "@cadl-lang/rest/http";
+
+export function transformPaths(model: Program): Paths {
```

Comment:
Can we move the `transformPaths` to the folder `transform`? considering we have sereval files to do transformation.